packages/zosjobs/src/doc/input/IGetJobsParms.ts

```diff
@@ -14,7 +14,7 @@
  * @export
  * @interface IGetJobsParms
  */
-export interface IGetJobsParms {
+ export interface IGetJobsParms {
```

Comment:
There appears to be a linting warning for the extra space on this line.

Reply:
Fixed!